test/runtests.jl

```diff
@@ -150,7 +150,7 @@ function simplexMagic(NDISC::Int=20, NPOINTS::Int=3, checkFileName::AbstractStri
 
     sInterpValTest = readdlm(checkFileName)
 
-    testErr = sum(abs(sInterpVal-sInterpValTest))
+    testErr = sum(abs, sInterpVal-sInterpValTest)
```

Comment:
Nice! You are a Julia power user now.

Reply:
or at least I read the deprecation warnings

Reply:
True, but I think others would have done `sum(abs.(x))` instead of `sum(abs, x)`. Your way is about 6 times faster and uses way less memory according to my quick test.

Reply:
oh, cool! I didn't take the time to test the difference, but I guess it makes sense since it doesn't have to allocate an intermediate array.